docs/spec/spec.md

```diff
@@ -293,7 +293,7 @@ The following API reads the content of a given CSV file as a string array of arr
 public isolated function fileReadCsv(string path, int skipHeaders = 0) returns string[][]|Error;
 ```
 
-The following API writes given CSV content to a given file.
+The following API writes given CSV content to a given file. When writing a `Record[]` content to a CSV file, by default, headers will be written to the CSV file as unlike `string[][],` the order of record fields is not guaranteed.
```

Comment:
Is this the correct behaviour? Even if the user appends a CSV file? 

Do not only focus on fixing the issue. Make sure the entire story is correct by looking into all possible scenarios.

Reply:
Updated in [ce506eb](https://github.com/ballerina-platform/module-ballerina-io/pull/827/commits/ce506ebbeab7df81297917880763f577a5ded002)